configure.ac

```diff
@@ -174,6 +174,16 @@ AC_CACHE_CHECK(whether ld accepts --version-script, ac_cv_version_script,
         ac_cv_version_script=no
     fi])
 
+AC_ARG_ENABLE([embedded],
+	      [AS_HELP_STRING([--enable-embedded],
+			      [Enable embedded support @<:@default=no@:>@])
+	      ],
+	      [ac_asm_symver_support=0
+               icc_symver_hack=1
+               embedded=true],
+	      [embedded=false])
+AM_CONDITIONAL([EMBEDDED], [test x$debug = xtrue])
```

Comment:
If you're going to enable/disable multiple things based on embedding or not, you should probably at least list what those things are in the help message.

Also, is there a reason you don't just use `$enable_embedded`?